app/app.iml

```diff
@@ -102,39 +102,39 @@
     </content>
     <orderEntry type="jdk" jdkName="Android API 23 Platform" jdkType="Android SDK" />
```

Comment:
This I never understood either.  The only library I added was the pngj one.  All the others existed previously.  But every merge I tried caused a whole lot of merge conflicts with the order.  Don't know when it changed so radically, I have only tried to reduce it.
  

Will be pushing up a new version with the order (hopefully) fixed tonight.
Cheers

---- tzachi-dar wrote ----

>In app/app.iml:
>
>> @@ -102,39 +102,39 @@ > </content> > <orderEntry type="jdk" jdkName="Android API 23 Platform" jdkType="Android SDK" /> 
>
>why are all this changes needed?
>
>It seems that no file was added / removed only order of files, but still the files don't seem to be ordered by any order?
>
>—
>Reply to this email directly or view it on GitHub.￼
>